### ISSUE TITLE
Options --max-ambig is not working with version 4

### ISSUE THREAD

**Issue by EricFournier3** [cov-lineages/pangolin#422]
Hi,

Here is an example with a sequence having N content of 0.6. Normally, we should expect lineage to be `None` as in version 3

(pangolinV3) [foueri01@inspq.qc.ca@slbio00d TEST_PANGOLIN]$ pangolin --all-versions
pangolin: 3.1.20
pangolearn: 2022-03-22
constellations: v0.1.4
scorpio: 0.3.16
pango-designation used by pangoLEARN/Usher: v1.2.133
pango-designation aliases: 1.2.141
(pangolinV3) [foueri01@inspq.qc.ca@slbio00d TEST_PANGOLIN]$ pangolin LAV-U0050464.fasta   --max-ambig 0.3 --outfile V3/res_LAV-U0050464.tsv --tempdir temp --threads 30
=>
(pangolinV3) [foueri01@inspq.qc.ca@slbio00d TEST_PANGOLIN]$ cat V3/res_LAV-U0050464.tsv
taxon,lineage,conflict,ambiguity_score,scorpio_call,scorpio_support,scorpio_conflict,version,pangolin_version,pangoLEARN_version,pango_version,status,note
Canada/Qc-LAV-U0050464/2021,**None**,,,,,,PANGO-v1.2.133,3.1.20,2022-03-22,v1.2.133,fail,N_content:0.6

---------------------------------------------------------------------------------------------------------------------------------------------------------------
(pangolinV4) [foueri01@inspq.qc.ca@slbio00d TEST_PANGOLIN]$ pangolin --all-versions
pangolin: 4.0.4
pangolin-data: 1.2.133
constellations: v0.1.4
scorpio: 0.3.16
pangolin-assignment: v1.2.133

(pangolinV4) [foueri01@inspq.qc.ca@slbio00d TEST_PANGOLIN]$ pangolin LAV-U0050464.fasta  --analysis-mode fast **--max-ambig 0.3** --outfile V4/res_LAV-U0050464.tsv --tempdir temp --threads 30
=>
(pangolinV4) [foueri01@inspq.qc.ca@slbio00d TEST_PANGOLIN]$ cat V4/res_LAV-U0050464.tsv
taxon,lineage,conflict,ambiguity_score,scorpio_call,scorpio_support,scorpio_conflict,scorpio_notes,version,pangolin_version,scorpio_version,constellation_version,is_designated,qc_status,qc_notes,note
Canada/Qc-LAV-U0050464/2021,**B.1.617.2**,,,Delta (B.1.617.2-like),0.38,0.0,scorpio call: Alt alleles 5; Ref alleles 0; Amb alleles 8; Oth alleles 0,SCORPIO_v0.1.4,4.0.4,0.3.16,v0.1.4,False,fail,**Ambiguous_content:0.61**,scorpio called lineage B.1.617.2

Thank you
Eric

**Comment by aineniamh** [comment]
The desired behaviour in pangolin 4.0 is that max-ambiguity is the cut off for the inference method (pangoLEARN or UShER). Scorpio has explicit SNPs it checks for so it is still possible to get a VOC call through scorpio even if the sequence doesn't pass the max-ambiguity threshold.

**Comment by whottel** [comment]
Hi,

Is there a way to set a max ambig cutoff that would apply to scorpio as well?
On a recent run I had two sequences called as BA.3 with Ambiguous_content: 0.57 and 0.48.
When I checked the actual reads it looks like there is very little evidene that these are in fact BA.3, as the only mutatons on this list for BA.3 (https://github.com/cov-lineages/constellations/blob/main/constellations/definitions/cBA.3.json) called in these sequences are also mutations listed as characteristic of BA.2. These sequences also had mutations that are unique to BA.2. I am wondering if the differences in the "min_alt" rules between BA.3 and BA.2 is the reason why these were called BA.3 over BA.2 by scorpio. In this case I think it would make sense for scorpio to not overrule a call of "Unassigned" given the high number of ambiguous bases.

Pangolin version info below:
pangolin: 4.0.6
pangolin-data: 1.6
constellations: v0.1.8
scorpio: 0.3.17
pangolin-assignment: v1.6

Thanks,
Wes

**Comment by aineniamh** [comment]
I think the status of this issue has changed as we have removed the 'probable' BA.* constellations. This gives a more strict definition of what these constellations are and will only get assigned if the sequence in question passes these thresholds. We believe that even if your sequence has a high N content, these defining sites are sufficient to call the variants. If you'd personally rather not have that information, I suggest a simple post-pangolin filtering step (we report the N content in the output file). 
Going to close now as I believe this particular case has been resolved.
